indico/core/config.py

```diff
@@ -51,6 +51,7 @@
     'EXTERNAL_REGISTRATION_URL': None,
     'HELP_URL': 'https://learn.getindico.io',
     'FAILED_LOGIN_RATE_LIMIT': '5 per 15 minutes; 10 per day',
+    'SIGNUP_RATE_LIMIT': '3 per 2 minutes; 20 per day',
```

Comment:
```suggestion
    'SIGNUP_RATE_LIMIT': '3 per 10 minutes; 20 per day',
```

should be more than enough.

also this needs to to be documented in settings.rst (like i did for `FAILED_LOGIN_RATE_LIMIT`). you can copy that text and just adapt the parts that mention logging in etc